src/terrain/clients/notifications.clj

```diff
@@ -47,17 +47,28 @@
 (defn send-team-join-notification
   "Sends a notification indicating that a user has requested to join a team."
   [user-name user-email team-name admin message]
-  (try
-    (send-notification {:type "team"
-                        :user (:id admin)
-                        :subject (str user-name " has requested to join team \"" team-name "\"")
-                        :email true
-                        :email_template "team_join_request"
-                        :payload {:email_address (:email admin)
-                                  :requester_name user-name
-                                  :requester_email user-email
-                                  :requester_message message
-                                  :team_name team-name}})))
+  (send-notification {:type "team"
+                      :user (:id admin)
+                      :subject (str user-name " has requested to join team \"" team-name "\"")
+                      :email true
+                      :email_template "team_join_request"
+                      :payload {:email_address (:email admin)
+                                :requester_name user-name
+                                :requester_email user-email
+                                :requester_message message
+                                :team_name team-name}}))
```

Comment:
Heh, I realized when I was looking over the code that I had a `try` without a `catch`. There's no real reason to allow this endpoint to succeed if the notification fails, so I just removed the `try`.